fixtures/js-stubs/broadcast.tsx

```diff
@@ -1,10 +1,13 @@
-export function Broadcast(params = {}) {
+import type {Broadcast as BroadcastType} from 'sentry/types';
```

Comment:
do we really need to rename types? In this case, we should consider adding the `type` suffix in all our types.

Reply:
Yes, and I disagree with adding a Type suffix - since this is a special case that only happens in stubs, I don't think it is enough to make the case to use in the rest of the codebase (or at least not yet). Adding a type suffix would also mean adding Interface/Enum etc, else we end up with 1 off rules that are not consistent with the rest of the codebase

Reply:
👍